daemon/resize.go

```diff
@@ -37,5 +38,17 @@ func (daemon *Daemon) ContainerExecResize(name string, height, width int) error
 	if err != nil {
 		return err
 	}
-	return daemon.containerd.ResizeTerminal(context.Background(), ec.ContainerID, ec.ID, width, height)
+	// Wait for the exec session ready.
+	timeout := 10 * time.Second
```

Comment:
Not a huge fan of the timeout, I would prefer for the context to be passed as one of the argument, but I guess it can be done in a separate PR.